dumpling-overview.md

```diff
@@ -80,6 +80,69 @@ dumpling \
 > 1. `--sql` 选项暂时仅仅可用于导出 csv 的场景。
 >
 > 2. 这里需要在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
+>
+> 3. csv文件不区分`字符串`与`关键字`，如果导入 boolean 类型的 `true` 和 `false`，需要转换为`1`和`0`。
+
+### 输出文件
+
++ metadata：此文件包含 dump 的起始时间，以及 master binary log 的位置。
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    cat metadata
+    ```
+
+    ```shell
+    Started dump at: 2020-11-10 10:40:19
+    SHOW MASTER STATUS:
+            Log: tidb-binlog
+            Pos: 420747102018863124
+
+    Finished dump at: 2020-11-10 10:40:20
+    ```
+
++ {schema}-schema-create.sql：schema 创建 sql 文件
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    cat test-schema-create.sql
+    ```
+
+    ```shell
+    CREATE DATABASE `test` /*!40100 DEFAULT CHARACTER SET utf8mb4 */;
+    ```
+
++ {schema}.{table}-schema.sql：table 创建 sql 文件
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    cat test.t1-schema.sql
+    ```
+
+    ```shell
+    CREATE TABLE `t1` (
+      `id` int(11) DEFAULT NULL
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
+    ```
+
++ {schema}.{table}.{0001}.{sql|csv|parquet}：数据源文件
```

Comment:
Dumpling doesn't support parquet format and doesn't have a plan to support it as far as I know.